pkg/provider/azure_loadbalancer.go

```diff
@@ -2739,10 +2754,11 @@ func (az *Cloud) getExpectedLBRules(
 }
 
 // getDefaultLoadBalancingRulePropertiesFormat returns the loadbalancing rule for one port
+// TODO: (anujbansal) Check how to handle lb rule updates for pods
```

Comment:
lb rule needs update upon endpoint-slice create/delete. Are you referring to this scenario or something else here?